salt/apache/init.sls

```diff
@@ -33,6 +33,23 @@ apache2:
     - require:
       - pkg: apache2
 
+# Uploads custom Apache defaults page and configuration
+  file.managed:
+    - names: 
+      - /var/www/html/error_page.html:
+        - source: salt://apache/files/docs/error_page.html
```

Comment:
Suggest moving this from `files/docs` to just `files`, since `docs` is specific to ocp19.